sample/src/main/java/com/schibstedspain/android/rxpager/sample/datasource/DataSource.java

```diff
@@ -37,7 +39,7 @@ public DataSource() {
 
   public Observable<TokenPage<String>> getPage(String s) {
     return Observable.fromCallable(() -> getPage(s, pages.get(s)))
-        .delay(1, TimeUnit.SECONDS);
+        .delay(1, TimeUnit.SECONDS, Schedulers.io());
     // delay operates on Scheduler.computation()
```

Comment:
Why?

Reply:
the current delay with 2 params on Rx2 it's deprecated so I added the sceduler

Reply:
I would like to see the deprecation to understand the reason.

Delay used to work on computation Scheduler, and now you changed it to the io so it also changed the behaviour.
( You can look at the comment in the next line )

Reply:
I correct this, is not a deprecation its an RxLint issue. I change it to computation